README.md

```diff
@@ -19,12 +21,16 @@ Run `cargo add pinecone-sdk` to add the crate as a dependency, or add the line `
 ## Usage
 
 The `PineconeClient` class is the main point of entry into the Rust SDK. Parameters may either be directly passed in as `Options`, or read through environment variables as follows. More detail:
+
```

Comment:
The spam of line breaks in here seems like it's from my vscode format-on-save settings. I think adding the spaces in is helpful visually.